### DISCUSSION TITLE
NuGet restore failing on .NET 5 for Debian-based OSes

### DISCUSSION THREAD

**Discussion by MichaelSimons** [dotnet/dotnet-docker#3683] [Announcements]
# NuGet restore failing on .NET 5 for Debian-based OSes
 
.NET 5 restore is broken for Debian-based operating systems including Docker thus resulting in error messages like:
 
```
error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
```
 
Please follow the [NuGet Announcement](https://github.com/NuGet/Announcements/issues/49) for the latest details on the issue.

Feel free to discuss here or on NuGet/Home#10491.

**Comment by MichaelSimons** [comment]
We have published new images that mitigate these issues by using a pre-release version of the Debian ca-certificates package. Please see the [NuGet announcement](https://github.com/NuGet/Announcements/issues/49#issuecomment-768766265)  for more details.

**Comment by davidhenley** [comment]
@MichaelSimons 

```
manifest for mcr.microsoft.com/dotnet/sdk:5.0.102-ca-patch not found: manifest unknown: manifest tagged by "5.0.102-ca-patch" is not found
```

**Comment by richlander** [comment]
This works ... 

```
docker pull mcr.microsoft.com/dotnet/sdk:5.0.102-ca-patch-buster-slim
```

Looks like we incorrectly documented some tags that don't exist or didn't publish some tags we intended. We'll get that sorted out in the morning, but the example above should work well enough in any case.

We updated the table @ https://github.com/NuGet/Announcements/issues/49#issuecomment-768766265 to have correct tags. Thanks for the report.

**Comment by aekiratli** [comment]
Any update on this?

**Comment by MichaelSimons** [comment]
> Any update on this?

See earlier update https://github.com/dotnet/dotnet-docker/issues/2550#issuecomment-768771743.

**Comment by davidhenley** [comment]
Thanks @richlander that worked!

**Comment by MichaelSimons** [comment]
A new version of the Debian ca-certificates package has been published which fixes the certificates issue. The .NET Docker images have been republished with the updated package. Please see the [NuGet announcement](https://github.com/NuGet/Announcements/issues/49#issuecomment-768766265) for more details.
